devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -229,6 +263,14 @@ def setLevel(level, rate = null) {
 
 def configure() {
 	log.debug "configure()"
+
+	if (isAeotecNanoDimmer()) {
+		state.configured = false
```

Comment:
I think you should refactor this to remove `state.configured` and replace it with the two boolean configuration flags that you are using elsewhere.

Reply:
we need all these 3 configuration flags for the initial configuration and it would be less clear if we replace that with two checkings for individual parameters

Reply:
now that there's only one preference I think you can refactor this to remove the `configured` flag. `state.minDimmingLevel` and `settings.minDimmingLevel` should be enough.

Reply:
@ZWozniakS